src/Altinn.AccessManagement.Core/Resolvers/Altinn/AltinnResourceResolver.cs

```diff
@@ -0,0 +1,59 @@
+using Altinn.AccessManagement.Core.Resolvers.Extensions;
+using Altinn.AccessManagement.Core.Services.Interfaces;
+
+namespace Altinn.AccessManagement.Core.Resolvers;
+
+/// <summary>
+/// Resolves attributes for <see cref="Urn.Altinn.Resource"/> 
+/// </summary>
+public class AltinnResourceResolver : AttributeResolver
+{
+    private readonly IContextRetrievalService _contextRetrievalService;
+
+    /// <summary>
+    /// ctor
+    /// </summary>
+    public AltinnResourceResolver(IContextRetrievalService contextRetrievalService) : base(Urn.Altinn.Resource.String())
```

Comment:
Unsure if or why we need a resource attribute resolver.

The ServiceResource model we get from the ResourceList from the resource-registry provides all the information we need, including the resource.AuthorizationReference which is the "resolved" attribute array we are to use for the service, regardless of whether it is an App or a Resource.

The resource-registry "owns" those attribute types and are not something we can or plan to change now.

Reply:
Since we don't know the input (unless checked), we can get an org + appid or resourceregistryid therefore we added the resolver that allows to us to cherry-pick fields independently of input data.